### ISSUE TITLE
prefer-invoke-map does not consider varialble arguments

### ISSUE THREAD

**Issue by levithomason** [wix/eslint-plugin-lodash#128]
`_.invokeMap` cannot be used here, since, the argument to each call is different (index based).

```js
activate(values = []) {
  return _.map(this.neurons, (neuron, i) => neuron.activate(values[i]))
}
```

**Comment by ganimomer** [comment]
Thanks for reporting this! Fixed in `v2.2.4`.

**Comment by levithomason** [comment]
Confirmed, thanks much!
